.devcontainer/scripts/onCreateCommand.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+
+set -e
+
+# prebuild the repo, so it is ready for development
+./build.sh libs+clr -rc Release
```

Comment:
@safern if we add `+libs.pretest` here, will that leave me in a state where I can walk up to any `/tests` folder and type `dotnet build /t:build;test` without any other prereqs?

If so it's probably worth adding as I'm assuming `libs.pretest` is fast.

Reply:
@danmoseley `libs.pretest` is already part of the default subsets for `libs`, so no need to add it.

Reply:
So I will be able to build and run an individual test library (it doesn't need libs.tests built?)

Reply:
Correct with the given setup you will be able to run, `dotnet build /t:test` on a test project and it will work. `libs.tests` is not needed unless you want to run all tests or send to helix.

Reply:
nit: you can also do just a `dotnet test` after a `build.cmd/sh clr+libs`, doesn't need to be `dotnet build /t:test`.

Reply:
This command is very CLR/Libraries team specific. Would it make sense to build the entire repo instead of just those two subsets? (including host and packs)

Reply:
We can always add more to the build in the future, if this turns out to be valuable to more people.

Reply:
Sounds good. My hope is that in the future we can spend some time on optimizing the repo's root build (including all subsets but with just one selected runtime) which should make it a no-brainer to build the entire repo. More and more parts will be required over time anyway, i.e. we don't use the live built host today in libraries but we want to. Similar for other parts like the targeting pack produced in the "packs" subset.

/braindump end